src/main/java/org/spectrumauctions/sats/core/model/mrvm/MRVMNationalBidder.java

```diff
@@ -84,7 +83,7 @@ public BigDecimal gammaFactor(MRVMRegionsMap.Region r, Bundle<MRVMLicense> bundl
         for (MRVMRegionsMap.Region region : getWorld().getRegionsMap().getRegions()) {
             result.put(region, gamma);
         }
-        return result;
+        return ImmutableMap.copyOf(result);
```

Comment:
Same as above, no need to cast to ImmuatableMap, as _result_ will go out of scope and modifications to it by external users are not harmful.

Reply:
:thumbsup: